go/vt/mysqlctl/mysqld.go

```diff
@@ -1195,9 +1214,18 @@ func (mysqld *Mysqld) GetVersionComment(ctx context.Context) string {
 	return versionComment
 }
 
-// applyBinlogFile extracts a binary log file and applies it to MySQL. It is the equivalent of:
+// ApplyBinlogFile extracts a binary log file and applies it to MySQL. It is the equivalent of:
 // $ mysqlbinlog --include-gtids binlog.file | mysql
-func (mysqld *Mysqld) applyBinlogFile(binlogFile string, includeGTIDs mysql.GTIDSet) error {
+func (mysqld *Mysqld) ApplyBinlogFile(ctx context.Context, binlogFile string, restorePos mysql.Position) error {
```

Comment:
Changing this to `mysql.Position` makes it easier to serialize / deserialize and better matches how we use a GTID position in other places here.